api.planx.uk/pay/pay.ts

```diff
@@ -27,16 +29,38 @@ export async function makePaymentViaProxy(
     process.env[`GOV_UK_PAY_TOKEN_${localAuthority.toUpperCase()}`];
 
   if (!isSupported) {
-    return next({
-      status: 400,
-      message: `GOV.UK Pay is not enabled for this local authority`,
-    });
+    return next(
+      new ServerError({
+        message: "GOV.UK Pay is not enabled for this local authority",
+        status: 400,
+      })
+    );
   }
 
   const flowId = req.query?.flowId as string | undefined;
   const sessionId = req.query?.sessionId as string | undefined;
   const teamSlug = req.params.localAuthority;
 
+  if (!flowId || !sessionId || !teamSlug ) {
```

Comment:
Please see comment above on optional query params - this might be an overcorrection.